src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -220,4 +277,123 @@ private void saveModifications(NetworkStoreListener listener) {
+        if (busbarSection == null) {
+            throw new NetworkModificationException(BUSBAR_SECTION_NOT_FOUND, loadCreationInfos.getBusId());
+        }
+
+        // creating the disconnector
+        int newNode = nodeBreakerView.getMaximumNodeIndex();
+        nodeBreakerView.newSwitch()
+                .setId("disconnector_" + loadCreationInfos.getEquipmentId())
+                .setName("disconnector_" + loadCreationInfos.getEquipmentName())
+                .setKind(SwitchKind.DISCONNECTOR)
+                .setRetained(false)
+                .setOpen(false)
+                .setFictitious(false)
+                .setNode1(busbarSection.getTerminal().getNodeBreakerView().getNode())
+                .setNode2(newNode + 1)
+                .add();
+
+        // creating the breaker
+        nodeBreakerView.newSwitch()
+            .setId("breaker_" + loadCreationInfos.getEquipmentId())
+            .setName("breaker_" + loadCreationInfos.getEquipmentName())
+            .setKind(SwitchKind.BREAKER)
+            .setRetained(false)
+            .setOpen(false)
+            .setFictitious(false)
+            .setNode1(newNode + 1)
+            .setNode2(newNode + 2)
+            .add();
+
+        // creating the load
+        voltageLevel.newLoad()
+            .setId(loadCreationInfos.getEquipmentId())
+            .setName(loadCreationInfos.getEquipmentName())
+            .setLoadType(loadCreationInfos.getLoadType())
+            .setNode(newNode + 2)
+            .setP0(loadCreationInfos.getActivePower())
+            .setQ0(loadCreationInfos.getReactivePower())
+            .add();
+    }
+
+    private void createLoadInBusBreaker(VoltageLevel voltageLevel, LoadCreationInfos loadCreationInfos) {
+        // busId is a bus id
+        VoltageLevel.BusBreakerView busBreakerView = voltageLevel.getBusBreakerView();
+        Bus bus = busBreakerView.getBus(loadCreationInfos.getBusId());
+        if (bus == null) {
+            throw new NetworkModificationException(BUS_NOT_FOUND, loadCreationInfos.getBusId());
+        }
+
+        // creating the load
+        voltageLevel.newLoad()
+            .setId(loadCreationInfos.getEquipmentId())
+            .setName(loadCreationInfos.getEquipmentName())
+            .setLoadType(loadCreationInfos.getLoadType())
+            .setBus(bus.getId())
+            .setConnectableBus(bus.getId())
+            .setP0(loadCreationInfos.getActivePower())
+            .setQ0(loadCreationInfos.getReactivePower())
+            .add();
+    }
+
+    public Flux<EquipmenModificationInfos> createLoad(UUID networkUuid, UUID groupUuid, LoadCreationInfos loadCreationInfos) {
+        return assertLoadCreationInfosNotEmpty(loadCreationInfos).thenMany(
+                getNetwork(networkUuid).flatMapIterable(network -> {
+                    NetworkStoreListener listener = NetworkStoreListener.create(network, groupUuid, modificationRepository);
```

Comment:
I think you mix here 2 concepts: listener et modification list.
The storage is done synchronously with `listener.storeLoadCreation(loadCreationInfos)` but you still use the concept of `NetworkStoreListener` which is not implemented for creation.
Add a `NetworkModificationList` which can be used inside a `NetworkStoreListener` or standalone like here.

Reply:
Will be done in another US